### PR TITLE
[LETS-546] Passive Transaction Server to Page Server communication request for MVCC status

### PR DESCRIPTION
**Pull request by wrlawodms**
http://jira.cubrid.org/browse/LETS-546

### CODE REVIEW

**Review by hornetmj — APPROVED** [review]

**Review by joohok — APPROVED** [review]

**Review by cristiarg — APPROVED** [review]
Please add relevant commit message. Not just the Jira link.

**Comment by wrlawodms** [comment]
@cristiarg Ok, I'll leave a commit message even if the PR description doesn't have 'cause it's trivial or the Jira issue's description is enough.

File: src/server/page_server.cpp [hunk 1]
```diff
@@ -181,6 +185,14 @@ page_server::connection_handler::receive_stop_log_prior_dispatch (tran_server_co
   m_conn->respond (std::move (a_sp));
 }
 
+void
+page_server::connection_handler::receive_oldest_active_mvccid (tran_server_conn_t::sequenced_payload &a_sp)
+{
+  /* TODO dummy function. will be modified corretly. soon  */
```
Comment (cristiarg) [inline src/server/page_server.cpp:191]: maybe, `assert (m_server_type == transaction_server_type::PASSIVE);` ?

**Review by cristiarg — APPROVED** [review]

**Comment by cristiarg** [comment]
> @cristiarg Ok, I'll leave a commit message even if the PR description doesn't have 'cause it's trivial or the Jira issue's description is enough.

@wrlawodms JIRA references can come & go; maybe at some point another issue system will be used, but the commit message stays with the code in the repo :)

**Comment by wrlawodms** [comment]
>@wrlawodms JIRA references can come & go; maybe at some point another issue system will be used, but the commit message stays with the code in the repo :)

I agree. Thank you for the advice!

File: src/server/page_server.cpp (hunk 1 above)
Comment (wrlawodms) [reply src/server/page_server.cpp:191]: Opps, already merged. I'll consider this a following PR, maybe in http://jira.cubrid.org/browse/LETS-548.

**Pull request merged**